tools/tr2/config/TR2X_ConfigTool/Resources/specification.json

```diff
@@ -62,6 +62,11 @@
           "Field": "enable_console",
           "DataType": "Bool",
           "DefaultValue": true
+        },
+        {
```

Comment:
Could we move this into `gameplay_options` below to match with TR1?

Reply:
@lahm86 addressed